Monika After Story/game/script-topics.rpy

```diff
@@ -4910,3 +4910,287 @@ label monika_asks_family:
+                                                                    m 4d "He wanted to push his students towards greatness."
+                                                                    m 4p "It's up to you if you think that's a good enough reason."
+                                                        "Nope, still nothing.":
+                                                            m 1p "I guess you forgot huh?"
+                                                            m 3m "The movie is pretty old."
+                                                            m 4n "Maybe it's been a while since you watched it."
+                                            m 1h "As for me, I can understand him."
+                                            m 2e "Surprised?"
+                                            m 2p "I know his methods were {i}extreme{/i} to say the least."
+                                            m 4o "But so was mine, right?"
+                                            m 4h "We both have our reasons too."
+                                            m 1c "So I guess I can understand him somewhat."
+                                            m 4l "But there's {i}no way{/i} I can ever hurl a chair at someone! Ahaha."
+                                            jump monika_watches_whiplash_end
+                                        "Those were just lucky guesses.":
+                                            m 1l "Well, I guess you managed to fool me [player]~"
+                                            m 4a "So that means you haven't watched it yet huh."
+                                            jump monika_watches_whiplash_synopsis
+                                "Neiman":
+                                    m 2l "Wrong! Caught you [player]!"
+                                    m 1j "Ahaha!"
+                                    jump monika_watches_whiplash_synopsis
+                        "Nicole":
+                            m 2l "Wrong! Caught you [player]!"
+                            m 1j "Ahaha!"
+                            jump monika_watches_whiplash_synopsis
+                        "Terrance":
+                            m 2l "Wrong! Caught you [player]!"
+                            m 1j "Ahaha!"
+                            jump monika_watches_whiplash_synopsis
+                "I was just kidding.":
+                    m 2l "Well, you could've fooled me [player]"
+                    m "Ahaha."
+                    jump monika_watches_whiplash_synopsis
+        "No.":
+            m 4b"It's a pretty good movie."
+            m 2b "Not my favourite, but if was nice."
+            if renpy.seen_label('monika_jazz'):
+                m 4b "And remember when we were talking about jazz? This is a jazz movie."
+            jump monika_watches_whiplash_synopsis
+            
+label: monika_watches_whiplash_synopsis:
```

Comment:
When you define a label you don't need to add `:` to it 
`label monika_watches_whiplash_synopsis:`

Reply:
done